packages/vscode-extension/vite.config.js

```diff
@@ -59,6 +59,7 @@ export default defineConfig({
           return "assets/[name]-[hash][extname]";
         },
       },
+      external: ["react-scan"],
```

Comment:
I don't know much about `vite`/`rollup`, but won't this cause the `react-scan` code to not be included in the final bundle, breaking the features that try to import and use it?